README.md

```diff
@@ -1,17 +1,21 @@
+# Announcing the new Azure SDK for JavaScript
 
-# **WARNING**
-- _All_ the management plane `azure-arm-*` sdks and _most_ of the data plane `azure-*` sdks, along with the rollup package [azure](https://www.npmjs.com/package/azure) in this repository, will be deprecated in **July, 2019**. They will be replaced by JS sdks in [azure-sdk-for-js](https://github.com/azure/azure-sdk-for-js).
-- The packages in [azure-sdk-for-js](https://github.com/azure/azure-sdk-for-js/tree/master/sdk) are isomorphic (can be run in the browser and on the server (node.js)) with accurate TypeScript type definitions.
-- They will be actively supported moving forward.
-- You will find any equivalent JS package in `@azure` org on npm for the existing node sdk package. For example: [azure-arm-compute](https://www.npmjs.com/package/azure-arm-compute) node sdk's equivalent JS sdk is [@azure/arm-compute](https://www.npmjs.com/package/@azure/arm-compute). 
-- _All_ the management plane sdks for the V1 Azure API (Azure Service Management) `azure-asm-*` will still be maintained from this repository. We would expect customers using these packages to move to Azure Resource Manager API packages at the earliest.
+We are excited to announce that we are building a new [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js), which includes both Node.js and Browser support! 
```

Comment:
This here doesn't tell me that the new sdk is being built elsewhere until I click on the link or read the next couple of lines
Text like this is also used to announce a new project in the repo where the readme lives.

Consider including a line on how the new project replaces the one in this repo?